docs/manual/src/proc_macro/docstrings.md

```diff
@@ -0,0 +1,14 @@
+# Docstrings
+
+In proc-macros, Rust docstrings will be captured and rendered in the bindings.
+
+For example:
+```rust
+/// This is the docstring for MyObject
+#[derive(uniffi::Object)]
+pub struct MyObject {}
+```
+
+Will cause Python, Swift and Kotlin to all generate a wrapper for `MyObject` with appropriate docstrings for that language.
+
+You can see examples of how they are rendered in the [UDL docstrings documentation](../udl/docstrings.md)
```

Comment:
I'm torn on this one.  I think "inlining" the UDL and proc-macro sections is nice when they're short, but if they got too long and went into too much specifics then they could be a distraction from the rest of the documents.

Where are these pages linked from?  In general it seems like there's a generic `types/foo.md` page that links to the `udl/foo.md` and `proc-macro/foo.md` pages.  If so, then it's hard to get to the specific pages without going through the general "why" / "how" sections.   But, that's not true for docstrings.

I'll just re-iterate that I'm torn.  Maybe we could inline the UDL/proc-macro sections in general, but use UDL/proc-macro specific pages when the text gets long.  I'd also fine with the current layout, it's really hard to know what's best.

Reply:
It already is that for custom types :) I do see the `types` version being the only version of the docs in many cases, and I actually agree docstrings might be a good candidate.

Reply:
Another potential solution for longer docs is to put more detail in the docstings themselves (with docstrings being the one we can't do that for...). This would mean these markdown docs have the most important/common information on usage and then link to docs.rs for more detailed information on specific objects. This also has the added benefit of giving instant access  to this information within IDEs. Although now that I think about it this only applies to proc macros and not UDL, so perhaps not a great solution.